docs/zh/user-guide/settings/managing-skills.md

```
@@ -382,13 +447,15 @@ spec:
 ### 1. 创建一个简单的 Skill
 
 创建目录结构:
+
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)